src/server/apis/org-config.ts

```diff
@@ -0,0 +1,63 @@
+import * as express from 'express';
+
+import {OrgSettings} from '../../types/api';
+import {GenericStatusResponse} from '../../types/api';
+import {settingsModel} from '../models/settingsModel';
+import {UserRecord} from '../models/userModel';
+
+import {PrivateAPIRouter} from './api-router/private-api-router';
+import * as responseHelper from './api-router/response-helper';
+
+export async function handleGetConfigRequest(request: express.Request) {
+  const orgName = request.params.orgName;
+  if (!orgName) {
+    return responseHelper.error(
+        'no-org-name', 'You must provide an \'orgName\' to use this API.');
+  }
+
+  const details = await settingsModel.getOrgSettings(orgName);
+  return responseHelper.data<OrgSettings|null>(details);
+}
+
+export async function handleSaveConfigRequest(
+    request: express.Request, userRecord: UserRecord) {
+  const orgName = request.params.orgName;
+  const newSettings = request.body.settings;
+  if (!orgName) {
+    return responseHelper.error(
+        'no-org-name',
+        'This API requires the \'orgName\' field to be supplied.');
+  }
+
+  if (!newSettings) {
+    return responseHelper.error(
+        'no-settings',
+        'This API requires the \'settings\' field to be a new configuration.');
+  }
+
+  if (typeof newSettings !== 'string') {
+    return responseHelper.error(
+        'settings-not-a-string',
+        'This API expects the \'settings\' api to be a plain text string.');
+  }
+
+  try {
+    await settingsModel.setOrgSettings(
```

Comment:
We need to do authorization checks. Currently this API would let anyone with a valid login to modify any org's settings. Instead, we'll need to do a GitHub API authorization check before each read/write.